apps/admin-gui/src/assets/i18n/en.json

```diff
@@ -1243,23 +1243,23 @@
       "EDIT_HINT": "You don't have permission to edit notifications.",
       "BASIC_SETTINGS": "Basic settings",
       "EMAIL_TYPE": "Email type",
-      "EMAIL_TYPE_DESCRIPTION_CREATE": "Selected type of notification (action which trigger sending and who is notified).",
-      "EMAIL_TYPE_DESCRIPTION_EDIT": "Type of notification (action which trigger sending and who is notified).",
+      "EMAIL_TYPE_DESCRIPTION_CREATE": "Selected type of notification (action which triggers sending and who is notified).",
+      "EMAIL_TYPE_DESCRIPTION_EDIT": "Type of notification (action which triggers sending and who is notified).",
       "APPLICATION_TYPE": "Application type",
```

Comment:
Info in parentheses is unclear and in my opinion could be structured better.

Reply:
unfortunately, this whole situation is unlucky with the use of the text in the context, 
![image](https://user-images.githubusercontent.com/32871167/141301691-b6a35bf7-1a0d-4d4a-9660-f0c42929a087.png)
I think this needs more like rework, and that's not part of the PR